src/app/slices/apiSlices/authSlice.js

```diff
@@ -2,7 +2,7 @@ import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
 
 export const authSlice = createApi({
   reducerPath: 'authApi',
-  baseQuery: fetchBaseQuery({ baseUrl: 'https://the-altcamp.onrender.com' }),
+  baseQuery: fetchBaseQuery({ baseUrl: 'https://api.thealtcamp.com' }),
```

Comment:
What do you think about importing `baseUrl` from `src/app/constants/api.js`?

That would make the code DRY. That way, subsequent changes to the URL will be made from the constants file

Reply:
This will be worked on. I'm trying to work on the layout change for the dashboard. Thanks for the suggestion. 🎉